Automattic/wp-calypso
# Login: add Akismet-branded login

## Proposed Changes

* Add an Akismet-branded login screen triggered by the `back` param in the redirect URL starting with `https://akismet.com`.

<img width="647" alt="Screenshot 2024-12-16 at 13 43 33" src="https://github.com/user-attachments/assets/413add5c-a10e-4ec1-9f1d-28b79fa22d58" />

## Why are these changes being made?

* The 'Sign in' link on akismet.com currently directs you straight to a 'Log in with WordPress.com' page with no indication that Akismet is involved.

## Testing Instructions

Go to http://calypso.localhost:3000/log-in?redirect_to=https%3A%2F%2Fr-login.wordpress.com%2Fremote-login.php%3Faction%3Dlink%26back%3Dhttps%253A%252F%252Fakismet.com%252Faccount%252F. This is the same URL generated by the 'Sign in' link on akismet.com.

Check that there is an Akismet logo, heading and green button as in the screenshot.

<img width="822" alt="Screenshot 2024-12-16 at 14 43 49" src="https://github.com/user-attachments/assets/3f5dbfd5-e70c-47c0-8f68-fa7927a5a95f" />

## Pre-merge Checklist
